README.md

```diff
@@ -1,53 +1,54 @@
-# Preparing the environment
-
-
-Follow the instructions here to install awestruct:
-https://awestruct.org/getting_started/
-
-Install NodeJS and libxml. NodeJS is required by the JavaScript compression module.
-```
-sudo apt-get install libxslt-dev libxml2-dev nodejs
-```
-Get the source code. Development code is stored on the develop branch.
-```
-git clone https://github.com/jbosstm/jbosstm.github.io
-cd jbosstm.github.io
-git checkout develop
-```
-
-Install required gems
-```
-bundle install
-```
-
-For more information about Awestruct refer https://awestruct.org
-
-# Web page preview
-
-If needed build and run a web page for a preview on a local server
-```
-awestruct -d
-```
-
-# Deploying web page
-
-Clean all unnecessary junk if required
-```
-git clean -d -f -x
-```
-
-Build deployable web page. This will generate the production web site.
-```
-awestruct -P production
-```
-
-Upload a web page to github.io. If everything went ok, web page will be updated automatically.
-```
-rsync -Pr --protocol=28 --delete-after _site/* jbosstm@filemgmt.jboss.org:www_htdocs/jbosstm/
-```
-
-
-If you did any changes on the development branch. Go back and push them to GitHub.
-```
-git push origin develop
-```
+# Preparing the environment
+
+Install awestruct (verified on Fedora 35):
+```sudo dnf -y update && sudo dnf -y install gcc-c++ make ruby-devel libxml2-devel libxslt-devel libffi-devel findutils git ruby tar redhat-rpm-config which python2 patchutils gem && dnf sudo clean all
+```
+
+Install NodeJS as it is required by the JavaScript compression module:
+```
+sudo dnf install nodejs
+```
+
+Clone this repository (or your fork). Development code is stored on the develop branch:
+```
+git clone https://github.com/jmfinelli/narayana.io.git
```

Comment:
Is it important to use a fork, if so I suggest to change `/jmfinelli/` to `/<forked_organisation>/` (or something like that), but if not then I think this should be `/jbosstm/`

Reply:
Thanks Tom, I have addressed your comment (my oversight)

Reply:
Thank you!